docs/SUMMARY.md

```diff
@@ -0,0 +1,60 @@
+# Table of contents
+
+* [Open Source Projects](README.md)
+* [clawback](clawback/README.md)
+  * [Clawback](clawback/clawback-of-issued-currency.md)
```

Comment:
Also, I'm not sure if 2-space indentations will work. The Markdown standard is 4 spaces.